packages/vuetify/src/components/VInput/VInput.tsx

```diff
@@ -129,14 +129,14 @@ export const VInput = genericComponent<new <T>() => {
         >
           { hasPrepend && (
             <div key="prepend" class="v-input__prepend">
+              { slots.prepend?.(slotProps.value) }
```

Comment:
Should we swap the order of append too for consistency?

Reply:
probably?